BUILD

```diff
@@ -1,3 +1,15 @@
-load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_prefix")
 
 go_prefix("istio.io/manager")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["config.pb.go"],
```

Comment:
Because I needed to add math/rand I ran `bin/gazelle` and it did this.  I will try removing it and see if everything still tests.